geode-core/src/main/java/org/apache/geode/distributed/internal/InternalDistributedSystem.java

```diff
@@ -318,7 +322,7 @@ public static InternalDistributedSystem connectInternal(Properties config,
    * disconnected
    */
   private final Set<DisconnectListener> disconnectListeners = new LinkedHashSet<>(); // needs to be
-                                                                                     // ordered
+  // ordered
```

Comment:
I would delete or move the end-of-line comments.

Reply:
Fixed in an upcoming commit.